product_secondary_unit/models/product_second_unit.py

```diff
@@ -22,7 +22,10 @@ class ProductSecondaryUnit(models.Model):
         help="Default Secondary Unit of Measure.",
     )
     factor = fields.Float(
-        string="Secondary Unit Factor", default=1.0, digits=0, required=True
+        string="Secondary Unit Factor",
+        default=1.0,
+        digits="Secondary Unit Factor",
```

Comment:
```suggestion
        digits="Product Unit of Measure",
```

Reply:
Yo do not need create a new one...

Reply:
I thought about it and looked at Odoo code - arguably this is a different thing.
`'Product Unit of Measure'` is used in a lot of places for product quantity precision.
This, on the other hand, is a conversion factor - which could easily be much more precise than what is needed for quantity fields in the whole system.

Reply:
But the unit of measure digits limit the conversion field resolution as well. If you have 2 decimal places for UoM, and try 0.001 as conversion factor, this is not going to be precisely expressed.

Reply:
That's a very good point.
You made me think about how the field is saved to a database and, to my horror, after this PR it's actually being just cut off (floored, I guess) at the specified precision, while before it was actually saved in any precision (even if not visible afterward in forms).
Now all the comments `# force NUMERIC with unlimited precision` make sense, where `digits=0` is also used. Strange, that this is not mentioned in the documentation for Float field.

The original idea of this PR is to allow a user to configure the visible precision for the factor field, as it really can, in specific use cases, be something like  `0.0624`.

How about, if I reimplement the same feature in XML views instead?

Reply:
I still don't see why putting a different precision from the product UoM. What do you think @sergio-teruel ?

Reply:
I agree with @pedrobaeza , sorry :cry: